client/src/App.tsx

```diff
@@ -28,6 +29,7 @@ function App(): JSX.Element {
                 <Route exact path="/signup" component={Signup} />
                 <Route exact path="/dashboard" component={Dashboard} />
                 <Route path="/profile/settings" component={Settings} />
+                <Route exact path="/manageBookings" component={ManageBookings} />
```

Comment:
```suggestion
                <Route exact path="/manage-bookings" component={ManageBookings} />
```
We use hyphenated case for routes names in JS